server/src/main/java/org/opensearch/cluster/coordination/UnsafeBootstrapClusterManagerCommand.java

```diff
@@ -101,6 +107,13 @@ protected boolean validateBeforeLock(Terminal terminal, Environment env) {
         if (clusterManager == false) {
             throw new OpenSearchException(NOT_CLUSTER_MANAGER_NODE_MSG);
         }
+        // During unsafe bootstrap, node will form a cluster with a new cluster UUID but with the existing metadata.
+        // This new state will not know about the previous cluster UUIDs and so we will not able to construct
+        // the cluster UUID chain to get the last known cluster UUID to restore from.
+        // Blocking unsafe-bootstrap below for this reason.
+        if (REMOTE_CLUSTER_STATE_ENABLED_SETTING.get(settings) == true) {
```

Comment:
should we also check node attrs ? to enable remote cluster state user needs to set both, cluster level settings and remote store node attrs.

Reply:
At the application level, we have check on both the enabled setting and the node attributes to ensure the remote state functionality works correctly. But I think we should keep this check simple.